connector/src/main/java/org/jboss/as/connector/subsystems/datasources/DataSourceDefinition.java

```diff
@@ -196,6 +196,23 @@ protected boolean isValueDiscardable(PathAddress address, String attributeName,
                 .end();
     }
 
+    static void registerTransformers130(ResourceTransformationDescriptionBuilder parentBuilder) {
+        ResourceTransformationDescriptionBuilder builder = parentBuilder.addChildResource(PATH_DATASOURCE);
+        builder.getAttributeBuilder()
+                .setDiscard(new DiscardAttributeChecker.DefaultDiscardAttributeChecker() {
+                    @Override
+                    protected boolean isValueDiscardable(PathAddress address, String attributeName, ModelNode attributeValue, TransformationContext context) {
+                        return attributeValue.equals(new ModelNode(false));
+                    }
+                }, TRACKING)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, TRACKING).end()
+                //We're rejecting operations when statistics-enabled=false, so let it through in the enable/disable ops which do not use that attribute
+                .addOperationTransformationOverride(DATASOURCE_ENABLE.getName())
+                .end()
+                .addOperationTransformationOverride(DATASOURCE_DISABLE.getName())
+                .end();
```

Comment:
@kabir can you explain what this does? I'm curious.

I suspect it doesn't apply here. The comment talks about "rejecting operations when statistics-enabled=false" but this 1.3.0 transformer isn't doing that. Doesn't need to since 1.3.0 understands that attribute.